docs/scripts/formattedTSDemos.js

```diff
@@ -88,7 +88,9 @@ async function transpileFile(tsxPath, project) {
     if (enableJSXPreview) {
       transformOptions.plugins = transformOptions.plugins.concat([
         [
-          require.resolve('docs/src/modules/utils/babel-plugin-jsx-preview'),
+          require.resolve(
+            'node_modules/@mui/monorepo/docs/src/modules/utils/babel-plugin-jsx-preview',
```

Comment:
This should work as well:

```suggestion
            '@mui/monorepo/docs/src/modules/utils/babel-plugin-jsx-preview',
```

Reply:
Yep that works, thanks!